### ISSUE TITLE
exporting json with attributes

### ISSUE THREAD

**Issue by mistatoms** [mbloch/mapshaper#180]
I am uploading to mapshaper:
https://github.com/charlesroper/OSGB_Grids/blob/master/Shapefile/OSGB_Grid_1km.shp
https://github.com/charlesroper/OSGB_Grids/blob/master/Shapefile/OSGB_Grid_1km.dbf

SHP and DBP combine nicely with PLAN_NO acting as key for shape attribute.
![ms](https://cloud.githubusercontent.com/assets/26797281/24512598/e69064a6-1566-11e7-91c5-42e849389039.jpg)

When exporting to GEOjson the attribute (dbf data) is stripped and powerBI cannot find the key to the polygons.

So I wonder: How can the json file be exported without losing the attribute?
If there is a script-to-export both GEOjson or TOPOjson including the attributes that would be fabulous!

**Comment by mbloch** [comment]
Hi, thanks for including links to your source data!

I wasn't able to reproduce the problem when I tested your files in the web interface and the command line interface. The console output (see below) indicates that the GeoJSON file contains the "PLAN_NO" field. When I exported to GeoJSON and opened the output file in mapshaper, the "PLAN_NO" field was also there.

The screenshot you included shows attribute data, so both the .shp and .dbf file were loaded successfully... is it possible that you exported to GeoJSON in a previous session in which only the .shp got loaded?

If that's not what happened, could you try loading your output GeoJSON back into mapshaper, to confirm that the attribute data is truly missing?

If this doesn't help, we'll have to do some more sleuthing to find out what happened.

```bash
$ mapshaper OSGB_Grid_1km.shp -o OSGB_Grid_1km.geojson
Wrote OSGB_Grid_1km.geojson

$ mapshaper OSGB_Grid_1km.geojson -info
Layer 1
Layer name: OSGB_Grid_1km
Records: 286,000
Geometry
  Type: polygon
  Bounds: -0.003792166454886923 -0.0007445552543941841 660000.0041139204 1224999.9966537368
  Proj.4: unknown
Attribute data
  Field    First value
  PLAN_NO  'HP4000'
```

**Comment by mistatoms** [comment]
Thanks a lot Matthew, this is really kind of you to trial my problem... I since managed to find and download from the same source the actual GeoJSON file.
https://github.com/charlesroper/OSGB_Grids/tree/master/GeoJSON

Then used mapper to translate it into TopoJSON which powerBI understands. That worked great.

Turns out that powerBI cannot handle such big JSON files and crashes (the 5km grids opened just fine although it struggled)... which maybe down to hardware restriction on my laptop. 

All i wanted was to colour the grids in three different colour according to the attributes from a CVS.
I have a friend who is fully ArcGIS enabled who managed to bake the CVS into the gson. But even he struggled with the number of shapes when loading it into a online viewer... surprisingly.

Thanks again for providing this awesome conversion tool which is doing such a neat job.
I will post a screenshot if i manage to solve the issue...
